docs: add new video and introduction page

AfganAbbas opened the pull request:
<!--- Provide a general summary of your changes in the title above -->

## Types of changes
<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)


## Description
<!--- Describe your changes in detail -->
<!--- Why is this change required? What problem does it solve? -->
<!--- If it resolves an open issue, please link to the issue here. For example "Resolves: #137" -->
Resolves: #333 
I added a new page(Introduction) to the documentation, according to other similar documentation websites, put video and key features into there

Update: I replaced video with the new one

## Checklist:
<!--- Put an `x` in all the boxes that apply. -->
<!--- If your change requires a documentation PR, please link it appropriately -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes (if not applicable, please state why)

Baroshem commented:
Hey there,

Thanks for writing the additional docs. However, I would prefer to only add a video as a part of this task (specifically replace the other video from Vue Nation as the one in the description better summarizes what the module does :))

I would also prefer to keep the documentation structure in line with other popular modules like nuxt image or nuxt content:
<img width="247" alt="image" src="https://github.com/Baroshem/nuxt-security/assets/37120330/22821eb4-641e-4172-ace8-d93005e0a38a">
<img width="288" alt="image" src="https://github.com/Baroshem/nuxt-security/assets/37120330/b18f41ec-1c3e-4971-b4ee-63e5fea727a3">

AfganAbbas commented:
Thanks for the information, I didn't understand the task quite well at first. I updated the branch, removed the added page(Introduction page in docs), and replaced the video

Baroshem commented:
No worries @AfganAbbas 

Sorry from my side for a really short description. I will add better descriptions for the next issues :)
